### PR TITLE
Adds context menu support for KDE

### PR DESCRIPTION
**Pull request by uspasojevic96**

### CODE REVIEW

**Review by christianparpart — CHANGES_REQUESTED** [review]
only minors, apart from that LGTM :)

File: Changelog.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
 - Fixes Sixel background select to support transparency (#450).
 - Fixes session resuming on KDE desktop envionment which is respawing all Contour instances upon re-login but failed due to invalid command line parameters (#461).
 - Changes DECSDM such that it works like a real VT340; also xterm, as of version 369, changed that recently (#287).
+- Adds context menu support for KDE
```
[the remark this answers is not in the document]
Comment (uspasojevic96) [reply Changelog.md]: done

File: debian/contour.install [hunk 2]
```diff
@@ -1,5 +1,6 @@
 debian/tmp/usr/bin/contour
 debian/tmp/usr/share/applications/contour.desktop
+debian/tmp/usr/share/kservices5/ServiceMenus/contour-run.desktop
```
[the remark this answers is not in the document]
Comment (uspasojevic96) [reply debian/contour.install]: done

**Review by christianparpart — APPROVED** [review]
Thanks!

**Pull request merged**